google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddShoppingProductAd.java

```diff
@@ -237,11 +237,11 @@ private String addStandardShoppingCampaign(
             // the ads from immediately serving. Set to ENABLED once you've added
             // targeting and the ads are ready to serve.
             .setStatus(CampaignStatus.PAUSED)
-            // Sets the bidding strategy to Manual CPC (with eCPC enabled)
+            // Sets the bidding strategy to Manual CPC (with eCPC disabled)
```

Comment:
Could you add a comment here explaining that if they set it to `true`, Google Ads will ignore the setting and the campaign will behave as if the setting was `false`, and link to the HC or blog post explaining why?

Reply:
Sure! Added suggested comments in a new commit.